src/main/java/net/minecraftforge/common/ForgeModContainer.java

```diff
@@ -120,6 +120,7 @@
     public static boolean logCascadingWorldGeneration = true; // see Chunk#logCascadingWorldGeneration()
     public static boolean fixVanillaCascading = false; // There are various places in vanilla that cause cascading worldgen. Enabling this WILL change where blocks are placed to prevent this.
                                                        // DO NOT contact Forge about worldgen not 'matching' vanilla if this flag is set.
+    public static List<String> unificationRegistryPriorityModIds = new ArrayList<>();
```

Comment:
`final`?

Reply:
Not really. I think this is subject to change for it's modified by the config.

Reply:
I will fix this so it's not trivially modifiable by mods.